meshinfo/network.py

```diff
@@ -108,15 +114,20 @@ async def reverse_dns_lookup(
 
     """
     with bound_contextvars(ip_address=ip_address):
+        logger.debug("Reverse DNS lookup", dns_server=dns_server)
         loop = asyncio.get_running_loop()
         on_con_lost = loop.create_future()
-        transport, protocol = await loop.create_datagram_endpoint(
+        transport, _protocol = await loop.create_datagram_endpoint(
             lambda: _DnsClientProtocol(ip_address, on_con_lost),
             remote_addr=(dns_server, 53),
         )
 
         try:
-            response = await on_con_lost
+            # There was weird issues with the poller hanging, so I addd a timeout here
+            # in case that was the issue.  I think a simultaneous upgrade to aiohttp
+            # might have fixed the issue, but I'm leaving this in for good measure.
```

Comment:
I need to revise this comment (and test a little more). Given that HTTPX failed to make a difference, I think it more likely that this was the fix, the error/exception just isn't propagating as I'd expect, so it's just a DNS error.

Reply:
I'm going to leave this in here, because testing the latest code for about a day without the `timeout()` didn't evidence an issue (and it was only working for about 4 hours previously).